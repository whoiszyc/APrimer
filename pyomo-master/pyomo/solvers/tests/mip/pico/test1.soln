LP solution:
Objective value = 3
X11 = 1
X23 = 1
X32 = 1
Dual solution:
NODEA1 = 1
NODEA2 = 1
NODEA3 = 1
