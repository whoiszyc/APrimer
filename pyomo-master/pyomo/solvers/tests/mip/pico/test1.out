Solver command line: /home/wehart/src/acro-utilib/bin/timer /Net/proj/opt/bin/LINUX/PICO --debug 2  --output /home/wehart/src/acro-utilib/packages/pyomo/test/plugins/mip/tmpGaCZjAtest1.soln /home/wehart/src/acro-utilib/packages/pyomo/test/plugins/mip/test1.mps

Solving problem: TINYMATCHING
Optimization Sense: minimization
MILPProblem Statistics
Number of variables:                 9
Number of binary variables:          0
Number of integer variables:         0
Number of general integer variables: 0
Number of special ordered sets:      0
Number of constraints:               3
Number of nonzeros:                  9
No integer variables... solving the LP normally.
	LP value= 3
	CPU RunTime=         0.001
	CPU TotalTime=       0.010998
	WallClock TotalTime= 0.0103740660707
real	        0.02
user	        0.01
sys	        0.00

